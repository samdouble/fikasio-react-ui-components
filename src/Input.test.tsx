import React from 'react';
import renderer from 'react-test-renderer';
import { Input } from '.';

describe('Input', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(<Input />).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
