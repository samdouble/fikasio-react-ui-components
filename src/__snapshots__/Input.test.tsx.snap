// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Renders correctly 1`] = `
<body>
  <div>
    <div
      class="fikasio-input fikasio-theme-dark"
      style="border: 1px solid #cccccc;"
    >
      <input
        class="form-control"
        style="border-radius: 0; height: 36px; padding: 6px 6px 6px 10px; width: 98%;"
        value=""
      />
    </div>
  </div>
</body>
`;
